stable/mysqldump/values.yaml

```diff
@@ -29,6 +37,9 @@ options: "--opt --single-transaction"
 # save sql backup to a directory named like the database or "alldatabases"
 saveToDirectory: false
 
+# if to dump all databases to stdout when not uploading
+dumpAll: false
```

Comment:
could we rename this to "dumpAllToStdout" please?

by the way, whats the usecase?
backup to elasticsearch or stackdirver via logging? :D

Reply:
Sure. :) It did the dump unconditionally previously. I found that unexpected and unnecessary. Getting some files on the PV is enough to see that it works.

Reply:
I should've split the cleanup stuff into separate commits and/or separate PR. Sorry about that.